.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug Report
+about: Create a report to help us improve rippled
+title: "[Title with short description] (Version: [rippled version])"
+labels: ''
+assignees: ''
+
+---
+<!-- Please search existing issues to avoid creating duplicates.-->
+<!-- If you are using a formal release, please use the version returned by './rippled --version' as the verison number-->
+<!-- If you are working off of develop, please add the git hash via 'git rev-parse HEAD'-->
+
+## Issue Description
+<!--Describe your bug/feature request in detail.-->
+
+## Steps to Reproduce
+<!--List in detail the exact steps to reproduce the unexpected behavior of the software.-->
+
+## Expected Result
+<!--Explain in detail what behavior you expected to happen.-->
+
+## Actual Result
+<!--Explain in detail what behavior actually happened.-->
+
+## rippled version
```

Comment:
I would remove this section and just include it as part of the Environment. The fewer sections the better, I think. Less intimidating to users.

Reply:
Will do